quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/Environment.java

```diff
@@ -141,27 +128,27 @@ public static String getProfileOrDefault(String defaultProfile) {
     }
 
     public static boolean isDevMode() {
-        if (DEV_PROFILE_VALUE.equalsIgnoreCase(getProfile())) {
+        if (org.keycloak.common.util.Environment.isDevMode()) {
             return true;
         }
```

Comment:
```suggestion
        return org.keycloak.common.util.Environment.isDevMode();
```

Reply:
This would break the fallback code below

Reply:
Ouch, right. Sorry, got confused. :)